csharp/src/Google.Protobuf.Test/CodedInputStreamTest.cs

```diff
@@ -119,12 +119,12 @@ private static void AssertReadVarint(byte[] data, ulong value)
         private static void AssertReadVarintFailure(InvalidProtocolBufferException expected, byte[] data)
         {
             CodedInputStream input = new CodedInputStream(data);
-            var exception = Assert.Throws<InvalidProtocolBufferException>(() => input.ReadRawVarint32());
-            Assert.AreEqual(expected.Message, exception.Message);
+            InvalidProtocolBufferException? exception = Assert.Throws<InvalidProtocolBufferException>(() => input.ReadRawVarint32());
```

Comment:
I suspect we can change this to be as it was before, just with `!` at the end of the `Throws` call. It really will be non-null, otherwise `Assert.Throws` will itself throw.